src/config/apis.ts

```diff
@@ -1,4 +1,4 @@
-const basePath = "http://localhost:4000/api";
+const basePath = "http://localhost:5000/api";
```

Comment:
Possibly create a `.env` file like we did on the front-end?

Reply:
Right now it isn't a task and it seemed easier and faster to just change a 4 to a 5